chai_research/.gitignore

```diff
@@ -1,4 +1,7 @@
 data
 scripts/train/explorations/logs
 *wandb*
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+*.env
+*__pycache__*
```

Comment:
Looks like this is not fulling taking effect? still some __pycache__ directories being added which we can prune.